developers.libra.org/docs/reference/libra-cli.md

```diff
@@ -18,9 +18,9 @@ To connect to the testnet through the CLI, a convenience script can be used to i
 ### Run a Local Libra Network and Spawn a CLI Client
 To start a local Libra network and spawn a CLI client that connects to this local network, run:
 ```bash
-cargo run -p libra-swarm -- -s
```

Comment:
Thanks for this great contribution1

These instructions are really old, we've been updating the docs to point to this new process: https://github.com/libra/libra/blob/master/developers.libra.org/docs/run-local-network.md

I'd be hesitant to add back swarm at the moment and would rather we move toward a unified process of using the `libra-node --test` and go from there.

Will leave the call up to @bmwill